### PR TITLE
puppetize config option for number of staff shirts, staff eligible for swag shirts

### PR DESCRIPTION
**Pull request by binary1230**
this is all part of the following:
https://github.com/magfest/production-config/pull/260
https://github.com/magfest/ubersystem-deploy/pull/83

https://github.com/magfest/ubersystem/pull/2776
https://github.com/magfest/ubersystem/pull/2777

### CODE REVIEW

File: templates/uber-development.ini.erb [hunk 1]
```diff
@@ -84,6 +84,11 @@ out_of_shirts = "<%= @out_of_shirts %>"
 shirts_per_staffer = <%= @shirts_per_staffer %>
```
Comment (kitsuta) [inline templates/uber-development.ini.erb]: Um...

File: manifests/config.pp [hunk 2]
```diff
@@ -131,6 +131,7 @@
   $shirts_per_staffer = 2,
```
Comment (kitsuta) [inline manifests/config.pp]: Er...

**Review by kitsuta — CHANGES_REQUESTED** [review]
shirts_per_staffer already existed in these two files. :) Please change their values rather than re-defining them!

**Comment by binary1230** [comment]
wooooooooooooooops totally didn't realize we just puppetized this, my bad.

I am still keeping this PR open though because I like the idea of leaving the default value undefined which means it'll be left off the INI which means the default in configspec.ini would be used.  trying to make it so the location of the default config lives mostly in configspec.ini and not in puppet, that way if we ever move away from puppet to something like docker or salt or whatever, the defaults live somewhere else.

**Comment by binary1230** [comment]
i made the changes here, will make the other PRs updated to not have the dupe stuff

**Review by kitsuta — APPROVED** [review]
+1!

I think actually we puppetized the option but never set it in our puppet config, so these are the only files where shirts_per_staffer existed.

**Pull request merged**